src/human_player_module.c

```diff
@@ -0,0 +1,391 @@
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <stdbool.h>
+#include "cards_management.h"
+#include "human_player_module.h"
+
+/* Global variables */
+bool g_card_requested = false;
+bool is_action_card = false;
+Player_t *human_player;
```

Comment:
We have defined Player_t g_players[PLAYERS_NUM]. I think we should not make redundant define here.